recipe/meta.yaml

```diff
@@ -30,15 +31,17 @@ requirements:
     - absl-py >=0.1.6
     - astor >=0.6.0
     - gast >=0.2.0
-    - numpy >=1.13.3
+    - numpy >=1.16.1
     - six >=1.10.0
     - protobuf >=3.6.0
-    - tensorboard 1.10.*
+    #- tensorboard 1.12.*  # NB: tensorboard is now bundled in the tensorflow wheel.
```

Comment:
tensorboard is a separate package on PyPI and in conda.  The tensorflow wheel does include the tensorboard entry point but tensorboard is still a dependency.